.env.example

```diff
@@ -120,15 +119,15 @@ LNBITS_KEY=LNBITS_ADMIN_KEY
 # LndWallet
 LND_GRPC_ENDPOINT=127.0.0.1
 LND_GRPC_PORT=10009
-LND_GRPC_CERT="/home/bob/.config/Zap/lnd/bitcoin/mainnet/wallet-1/data/chain/bitcoin/mainnet/tls.cert"
-LND_GRPC_MACAROON="/home/bob/.config/Zap/lnd/bitcoin/mainnet/wallet-1/data/chain/bitcoin/mainnet/admin.macaroon or HEXSTRING"
+LND_GRPC_CERT="/home/bob/.lnd/tls.cert"
+LND_GRPC_MACAROON="/home/bob/.lnd/data/chain/bitcoin/mainnet/admin.macaroon"  # or HEXSTRING
 # To use an AES-encrypted macaroon, set
 # LND_GRPC_MACAROON="eNcRyPtEdMaCaRoOn"
 
 # LndRestWallet
 LND_REST_ENDPOINT=https://127.0.0.1:8080/
-LND_REST_CERT="/home/bob/.config/Zap/lnd/bitcoin/mainnet/wallet-1/data/chain/bitcoin/mainnet/tls.cert"
-LND_REST_MACAROON="/home/bob/.config/Zap/lnd/bitcoin/mainnet/wallet-1/data/chain/bitcoin/mainnet/admin.macaroon or HEXSTRING"
+LND_GRPC_CERT="/home/bob/.lnd/tls.cert"
```

Comment:
i guess this should be `LND_REST_CERT` :)

Reply:
fixed in 7461f5505846bb69e3858f0957e03ed747ff847b